### PR TITLE
Fixes pkp/pkp-lib#4780, pkp/pkp-lib#4783 and pkp/pkp-lib#4784

### PR DESCRIPTION
**Pull request by jonasraoni**
Fixes pkp/pkp-lib#4780, pkp/pkp-lib#4783 and pkp/pkp-lib#4784 by respectively checking absolute URLs in the code that fixes relative URLs, creating the missing locale key and setting a "placeholder" for the role name.

### CODE REVIEW

File: controllers/grid/queries/form/QueryForm.inc.php [hunk 1]
```diff
@@ -28,7 +28,7 @@ class QueryForm extends Form {
 	/** @var Query The query being edited **/
 	var $_query;
 
-	/** @var boolean True iff this is a newly-created query */
```
Comment (asmecher) [inline controllers/grid/queries/form/QueryForm.inc.php]: "iff" is shorthand for "if and only if".

Comment (jonasraoni) [reply controllers/grid/queries/form/QueryForm.inc.php]: @asmecher Done (I recreated the commit)!

**Comment by jonasraoni** [comment]
@asmecher Done (I recreated the commit)!

**Comment by asmecher** [comment]
Merged, thanks, and partially back-ported to `stable-3_1_2`! Even though these are nearly trivial fixes, it's better to keep them in separate pull requests, so we can easily selectively cherry-pick them back/forward to other branches.

**Pull request merged**